src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
```

Comment:
The class 'Ball' could benefit from a constructor to initialize its fields 'color' and 'number'.